### PR TITLE
 Added "osm2pgsql" version of updating page.

### PR DESCRIPTION
**Pull request by SomeoneElseOSM**
Added "osm2pgsql" version of updating page.

Also removed links to 21.04 guide, updated copyright year,
renamed "osmosis" version of updating page to say "osmosis".

### CODE REVIEW

**Comment by SomeoneElseOSM** [comment]
Some comments from IRC: . in comma separated list, and "seet".

File: serving-tiles/manually-building-a-tile-server-20-04-lts.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ Then we convert the carto project into something that Mapnik can understand:
 You now have a Mapnik XML stylesheet at /home/renderaccount/src/openstreetmap-carto/mapnik.xml .
 # Loading data
 
-Initially, we'll load only a small amount of test data. Other download locations are available, but "download.geofabrik.de" has a wide range of options. In this example we'll download the data for Azerbaijan, which is about 17Mb.
+Initially, we'll load only a small amount of test data. Other download locations are available, but "download.geofabrik.de" has a wide range of options. In this example we'll download the data for Azerbaijan, which is about 32Mb.
 
 Browse to https://download.geofabrik.de/asia/azerbaijan.html and note the "This file was last modified" date (e.g. "2020-05-02T20:59:02Z"). We'll need that later if we want to update the database with people's susbsequent changes to OpenStreetMap. Download it as follows:
```
Comment (Andygol) [inline serving-tiles/manually-building-a-tile-server-20-04-lts.md:170]: The extract timestamp also needs to be updated
```suggestion
Browse to https://download.geofabrik.de/asia/azerbaijan.html and note the "This file was last modified" date (e.g. "2022-06-05T20:21:37Z"). We'll need that later if we want to update the database with people's susbsequent changes to OpenStreetMap. Download it as follows:
```

Comment (pnorman) [reply serving-tiles/manually-building-a-tile-server-20-04-lts.md:170]: Why does this need modifying?

Comment (SomeoneElseOSM) [reply serving-tiles/manually-building-a-tile-server-20-04-lts.md:170]: I changed the download size to match the current size, and andygol changed the date to match too...

Comment (Andygol) [reply serving-tiles/manually-building-a-tile-server-20-04-lts.md:170]: We need to adjust this date in all parts of the documentation, not only here

Comment (pnorman) [reply serving-tiles/manually-building-a-tile-server-20-04-lts.md:170]: The download size says it is what the file currently is, but this is an example that will never be what it currently is except for one day.

File: serving-tiles/manually-building-a-tile-server-debian-11.md [hunk 2]
```diff
@@ -115,7 +115,7 @@ Then we convert the carto project into something that Mapnik can understand:
 You now have a Mapnik XML stylesheet at /home/youraccountname/src/openstreetmap-carto/mapnik.xml .
 # Loading data
 
-Initially, we'll load only a small amount of test data. Other download locations are available, but "download.geofabrik.de" has a wide range of options. In this example we'll download the data for Azerbaijan, which is about 17Mb.
+Initially, we'll load only a small amount of test data. Other download locations are available, but "download.geofabrik.de" has a wide range of options. In this example we'll download the data for Azerbaijan, which is about 32Mb.
 
 Browse to https://download.geofabrik.de/asia/azerbaijan.html and note the "This file was last modified" date (e.g. "2020-11-13T21:42:03Z"). We'll need that later if we want to update the database with people's susbsequent changes to OpenStreetMap. Download it as follows:
```
Comment (Andygol) [inline serving-tiles/manually-building-a-tile-server-debian-11.md:120]:
```suggestion
Browse to https://download.geofabrik.de/asia/azerbaijan.html and note the "This file was last modified" date (e.g. "2022-06-05T20:21:37Z"). We'll need that later if we want to update the database with people's susbsequent changes to OpenStreetMap. Download it as follows:
```

File: serving-tiles/manually-building-a-tile-server-ubuntu-21.md [hunk 3]
```diff
@@ -110,7 +110,7 @@ Then we convert the carto project into something that Mapnik can understand:
 You now have a Mapnik XML stylesheet at /home/youruseraccount/src/openstreetmap-carto/mapnik.xml .
 # Loading data
 
-Initially, we'll load only a small amount of test data. Other download locations are available, but "download.geofabrik.de" has a wide range of options. In this example we'll download the data for Azerbaijan, which is about 17Mb.
+Initially, we'll load only a small amount of test data. Other download locations are available, but "download.geofabrik.de" has a wide range of options. In this example we'll download the data for Azerbaijan, which is about 32Mb.
 
 Browse to https://download.geofabrik.de/asia/azerbaijan.html and note the "This file was last modified" date (e.g. "2021-09-16T20:21:14Z"). We'll need that later if we want to update the database with people's susbsequent changes to OpenStreetMap. Download it as follows:
```
Comment (Andygol) [inline serving-tiles/manually-building-a-tile-server-ubuntu-21.md:115]:
```suggestion
Browse to https://download.geofabrik.de/asia/azerbaijan.html and note the "This file was last modified" date (e.g. "2022-06-05T20:21:37Z"). We'll need that later if we want to update the database with people's susbsequent changes to OpenStreetMap. Download it as follows:
```

File: serving-tiles/manually-building-a-tile-server-20-04-lts.md (hunk 1 above)
Comment (SomeoneElseOSM) [reply serving-tiles/manually-building-a-tile-server-20-04-lts.md:170]: Rereading https://switch2osm.org/serving-tiles/manually-building-a-tile-server-20-04-lts/ , it does say at the top "These step-by-step instructions were written for [Ubuntu Linux](https://ubuntu.com/) 20.04 LTS (Focal Fossa), and were tested in May 2020" so I guess that the example download date of 2020 does make sense 

I've added a "currently" to the file size to hopefully make things clearer and left the dates, which tend to match when each guide was written.

**Pull request merged**